dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -30,6 +29,8 @@
  *= require details-polyfill
  */
 
+// TODO: Move into code-studio.scss when it exists.
+@import "/code-studio/assets/video-js/video-js.css";
```

Comment:
Here's a [quick experiment with a custom SASS importer](https://github.com/code-dot-org/code-dot-org/compare/create-code-studio-scss) (a feature they claim is experimental, but it's been around since the last major version).  It seems to work great; unfortunately I discovered video.js is a poor candidate for using this, because their CSS file contains relative references to font files they need to load.